01-Introducing-Molecular-Dynamics/02b-Running-Rigid-Body-Molecular-Dynamics-System.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>Comment why we exclude <code>'body'</code>.  Also, just set the default <code>r_cut</code>  to 2.5</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>